src/selectors.js

```diff
@@ -33,15 +33,20 @@ export function getRelationship(state, entity, relationshipName) {
     return null;
   }
 
-  const { data } = entity.relationships[relationshipName];
+  if (!entity.relationships[relationshipName]) {
+    return null;
+  }
 
+  const { data } = entity.relationships[relationshipName];
```

Comment:
This is where an exception was being raised, when relationshipName did not exist on relationships